Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -679,7 +679,22 @@ def setup_(self, step, stepName, stepDict, k, properties):
     offset = 0.512,
 )
 
-# add here a .513 workflow for GPU vs CPU validation
+upgradeWFs['PatatrackECALOnlyGPUValidation'] = PatatrackWorkflow(
+    digi = {
+        '--accelerators': 'gpu-nvidia',
+        '--procModifiers': 'gpu'
+    },
+    reco = {
+        '-s': 'RAW2DIGI:RawToDigi_ecalOnly,RECO:reconstruction_ecalOnly,VALIDATION:@ecalOnlyValidation,DQM:@ecalOnly',
+        '--accelerators': 'gpu-nvidia',
+        '--procModifiers': 'gpuValidation'
```

Comment:
I think you still need the `gpu` modifier:
```suggestion
        '--procModifiers': 'gpu,gpuValidation'
```

Sorry for the confusion !

Reply:
Ah, no -- `gpuValidation` includes `gpu` already.